src/clj/server/csv.clj

```diff
@@ -0,0 +1,37 @@
+(ns server.csv
+  (:require
+   [clojure.string :as string]
+   [shoreleave.middleware.rpc :refer [defremote]]))
+
+(defremote csv-files
```

Comment:
>  think it makes sense to have one remote call for backing up data. Something that takes a file name and a sequence of data.

@joshcurtis So turn csv logic into generic file logic, and put the csv logic in clj?